scenes/menus/storybook/components/storybook.gd

```diff
@@ -51,12 +58,15 @@ func _enumerate_quests() -> Array[Quest]:
 
 
 func _ready() -> void:
+	animated_book.animation_finished.connect(_on_animation_finished)
+
 	var previous_button: Button = null
 	for quest in _enumerate_quests():
 		var button := Button.new()
 		button.text = quest.get_title()
 		button.theme_type_variation = "FlatButton"
 		quest_list.add_child(button)
+		button.set_meta("quest", quest)
```

Comment:
I've never seen `Object.set_meta` before!

Reply:
I haven't used it much, It is wild. Metadata can be anything. There is a `+ Add Metadata` button in the Inspector at the bottom of any node or resource you are inspecting.